ecjtu/ecjtu_api/respose_result.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Dict, Union
+
+from fastapi.responses import JSONResponse
+
+
+class ResponseResult:
+    @staticmethod
+    def success(
+        data: Union[Dict[str, Any], Any] = None, msg: str = "success"
```

Comment:
This makes me a little confused. Any means any type. It doesn't make sense to go with Union